### ISSUE TITLE
Is AQEMU development still alive?

### ISSUE THREAD

**Issue by EarthlingX** [tobimensch/aqemu#67]
Just out of curiosity: I would love to switch from `virt-manager` to `aqemu`. Is development here still alive? I would appreciate it!

**Comment by andreymal** [comment]
ping @tobimensch

**Comment by tobimensch** [comment]
Yes. And no. And yes. I was working on a much improved partly rewritten AQEMU version years back, but I had to put the work on hold. Today I merged some of the long standing pull requests on GitHub in good faith that they're hopefully well done. Distributions/Packagers of aqemu should definitely check out the latest master branch on GitHub, because I think it fixes some issues for people.

I'll definitely keep accepting pull requests in this same manner, and when I get around to merging all of this with the branch I worked out years ago, which is really really cool, aqemu will be lifted to the next level.

Also I was thinking about doing a complete Nim-rewrite of aqemu, but for this to be possible, there'd need to be a nim bindings for Qt first. Not sure if I'm going down this path.

I'm interested in why people want to use aqemu instead of virt-manager or VirtualBox? Aren't they both pretty good as well? It's not that I want to downplay aqemu, I'm just interested in what aqemu's user like about it over the alternatives?

**Comment by andreymal** [comment]
>I'm interested in why people want to use aqemu instead of virt-manager or VirtualBox?

For me:

1) AQEMU is a GUI for QEMU which has more features than VirtualBox

2) virt-manager requires the libvirt daemon, it's annoying

**Comment by EarthlingX** [comment]
> I'm interested in why people want to use aqemu instead of virt-manager or VirtualBox?

For me: AQEMU is written in Qt instead of GTK and therefore fits better to LXQt/KDE-Plasma/etc.

**Comment by tobimensch** [comment]
> For me: AQEMU is written in Qt instead of GTK and therefore fits better to LXQt/KDE-Plasma/etc.

I understand that well, I'm currently back to being a KDE user.

Still, I'm not in an eternal war to prove that Qt is better than GTK, so the question for me is, what can AQEMU do for users that other solutions can't, or won't.

**Comment by joergister** [comment]
Im no longer using Linux but what it did different for me back when I was using it on a Plasma desktop : it’s ui was easier to understand when coming from virtualbox and starting to use kvm and qemu than virtmanager was. It was a pleasant way to manage vms.

Some things were outdated and annoying to use like sharing local directory’s with the VM, that was much more challenging then it was on virtualbox. But I don’t know what that would have been like using virtmanager so that might just be a qemu problem since that was not really designed to be a desktop vm manager.

**Comment by Butterfly** [comment]
> I'm interested in why people want to use aqemu instead of virt-manager or VirtualBox? Aren't they both pretty good as well? It's not that I want to downplay aqemu, I'm just interested in what aqemu's user like about it over the alternatives?

VirtualBox was ok for me when I tried it a few times. I've never messed with `virt-manager` directly so I don't know about that. Test-driving `virt-manager` is actually on a second-tier personal #TODO list... just for funnsies.

What I like about `aqemu` is that it's available in the very most [basic "main" Debian repositories](http://ftp.debian.org/debian/pool/main/a/aqemu/). Absolute #1 for me: That inclusion implies a certain level of trustworthiness which is a win!

**Comment by xlucn** [comment]
I am quite a newbie user of `qemu`. I would appreciate a lightweight GUI front-end as a visual guidance to help me tweak my VM settings.
What `virt-manager` doesn't have: lightweight as in installation dependences.
What `VirtualBox` doesn't have: I am not sure. I think I have better performance in VirtualBox, but that is partially because I haven't tried the hundreds of options in `qemu` and I have no idea where to start. Thus if `aqemu` could help with that I will be glad to switch.
